jmaqs-webservices-jdk11/pom.xml

```diff
@@ -11,9 +11,52 @@
 
     <groupId>com.magenic.jmaqs.webservices</groupId>
     <artifactId>jmaqs-webservices-jdk11</artifactId>
-    <version>2.0.1-SNAPSHOT</version>
     <name>JMAQS JDK11 WebServices Testing Module</name>
+    <version>2.0.1-SNAPSHOT</version>
 
+    <dependencies>
+        <dependency>
+            <groupId>com.magenic.jmaqs</groupId>
+            <artifactId>jmaqs-framework</artifactId>
+            <version>2.0.1-SNAPSHOT</version>
+            <scope>compile</scope>
+        </dependency>
```

Comment:
This is your build failure.  2.0.1-SNAPSHOT doesn't currently exist out in the wild.

Reply:
Also, there isn't a reason to inherit the entire framework.  Technically, it is a pom that is inherited.

Reply:
I just saw that too. A typo on my part